_includes/enterprise-payer-reference.md

```diff
@@ -1,55 +1,39 @@
-|                  | └➔&nbsp;`payer`                    | `object`     | The `payer` object containing information about the payer relevant for the payment order.                                                                                                                                                                                                     |
-|   | └➔&nbsp;`requireConsumerInfo`                       | `bool` | Set to `true` if the merchant wants to receive profile information from Swedbank Pay. Applicable for when the merchant only needs `email` and/or `msisdn` for digital goods, or when the full shipping address is necessary. If set to `false`, Swedbank Pay will depend on the merchant to send `email` and/or `msisdn` for digital products and shipping address for physical orders. |
-| | └➔&nbsp;`digitalProducts`                       | `bool` | Set to `true` for merchants who only sell digital goods and only require `email` and/or `msisdn` as shipping details. Set to `false` if the merchant also sells physical goods. |
-| {% icon check %} | └─➔&nbsp;`firstName`                    | `string`     | The first name of the payer.                                                                                                                                                                                                                                                                              |
-| {% icon check %} | └─➔&nbsp;`lastName`                    | `string`     | The last name of the payer.                                                                                                                                                                                                                                                                              |
-|                  | └─➔&nbsp;`email`                   | `string`     | The e-mail address of the payer. Increases the chance for [frictionless 3-D Secure 2 flow][3d-secure-2].                                         |
-|                  | └─➔&nbsp;`msisdn`                  | `string`     | The mobile phone number of the payer. The mobile number must have a country code prefix and be 8 to 15 digits in length. The field is related to [3d-secure-2][3d-secure-2].            |
-|                  | └─➔&nbsp;`payerReference`                     | `string`     | A reference used in Enterprise and Payments integrations to recognize the payer in the absence of SSN and/or a secure login.                                                                                                                                                                                                                               |
-
-[3d-secure-2]: /checkout-v3/enterprise/features/core/3d-secure-2
-[enterprise-enter-ssn]: /assets/img/checkout/enterprise-enter-ssn.png
+[enterprise-enter-ssn]: /assets/img/checkout/enterprice-enter-ssn.png
```

Comment:
```suggestion
[enterprise-enter-ssn]: /assets/img/checkout/enterprise-enter-ssn.png
```

Haha!  Ain't winning no price for enterpriCe!

Reply:
:D